docker/web-and-data/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# This image includes both the web app and the data import related scripts in
```

Comment:
I cannot find what command you're using to build an image using this Dockerfile, giving it access to .dockerignore and the right folder to `COPY` in, while ignoring the other Dockerfile that's still in the root folder of the repo. I presume you're just building it from that root folder, but with an added `-f` argument to target this Dockerfile?

Reply:
yes that's how i was using it - im not super familiar with docker hub pipelines or circle ci pushing of images so have to figure out what directory structure will work for that. Might have to change it slightly, but i can add the command im currently using to this PR, maybe as a comment on this image

Reply:
added some usage instructions